src/libNetwork/P2PComm.h

```diff
@@ -51,6 +51,7 @@ class SendJob {
   unsigned char m_startbyte{};
   bytes m_message;
   bytes m_hash;
+  bool m_allowSendToRelaxedBlacklist{};
```

Comment:
set to atomic?

Reply:
I don't think we need that in this case. Its as good as other params only set once in
https://github.com/Zilliqa/Zilliqa/blob/7e37c8cfd771f85cb799226156848241c16f896c/src/libNetwork/P2PComm.cpp#L821
Thread safety is fine.